content/docs/command-reference/remote/modify.md

```diff
@@ -1202,16 +1202,26 @@ by HDFS. Read more about by expanding the WebHDFS section in
   1. `user` parameter set with this command (found in `.dvc/config`);
   2. User defined in the URL (e.g. `webdavs://user@example.com/endpoint/path`)
 
-- `password` - password for WebDAV server, can be empty in case of using `token`
-  authentication.
+- `custom_auth_header` - HTTP header field name to use for authentication. Value
+  is set via `password`.
+
+  ```cli
+  $ dvc remote modify --local myremote \
+                      custom_auth_header 'My-Header'
+  ```
+
+- `password` - password for WebDAV server, combined either with `user` or
+  `custom_auth_header`. Leave empty for `token` authentication.
 
   ```cli
   $ dvc remote modify --local myremote password mypassword
   ```
 
-> Note that `user/password` and `token` authentication are incompatible. You
-> should authenticate against your WebDAV remote by either `user/password` or
-> `token`.
+  <admon type="info">
+
+  `user/custom_auth_header`, `password` auth is incompatible with `token` auth.
```

Comment:
It is more about the combination, so I think it should be:
```suggestion
  `user/password`, `custom_auth_header/password` auth is incompatible with `token` auth.
```

Reply:
OK, I'm just not sure this wording is clear... 🤔 How about this:

```suggestion
  `password`-based auth (either with `user` or `custom_auth_header`) is incompatible with `token` auth.
```

Reply:
yeah, this one's a bit better, but I'd like to highlight `user` vs `custom_auth_header` vs `token` auth type more (`password` does not seem very important or is more obvious).